packages/cloud-config/oem/10_selinux.yaml

```diff
@@ -0,0 +1,16 @@
+# Default cOS OEM configuration file
```

Comment:
All this file is pretty tied to the immutable layout so I am wondering to what extend this should be part or not of the immutable-rootfs dracut module and include there an early boot systemd unit that does this thing if selinux is active (`selinux=1` kernel parameter or active in `/etc/selinux/config`). 

Probably we could apply the restorecon to any `rw` path and `rw` paths are managed by the immutable rootfs module. Anyway I do not want to modify immutable rootfs module code now, as changes there tend to strike back sooner or later.